### ISSUE TITLE
FBMS

### ISSUE THREAD

**Issue by yangshushuaige** [sabarim/3DC-Seg#4]
Hello, I now have the FBMS results (.png) predicted by my network, can you tell me how to calculate Fmean? I cannot find or use the Ground-Truth. Thank you very much and look forward to your reply.

**Comment by linhaoqi027** [comment]
hi ,do u known where to get pre-trained model named irCSN_152_ft_kinetics_from_ig65m_f126851907.pth in Base.yaml

**Comment by Ali2500** [comment]
I'm not sure what you mean here exactly, but the ground truth is available as part of the FBMS dataset: https://lmb.informatik.uni-freiburg.de/resources/datasets/

**Comment by colorblank** [comment]
I have same problem. The code you wrote loading test set of FBMS reads "*.png" as ground truth, but the official FBMS test set only has partial ground truth with "*.ppm" format. I'm not sure how the evaluation on FBMS made. Is the ".png" ground truth from FBMS  privately or just using train set for evaluation？
